### PR TITLE
feat: mail tempaltes/texts and step 10

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

File: internal/v2/domain/policy_mail_text.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import "github.com/caos/zitadel/internal/eventstore/models"
+
+type MailText struct {
+	models.ObjectRoot
+
+	State        PolicyState
+	Default      bool
+	MailTextType string
+	Language     string
```
Comment (livio-a) [inline internal/v2/domain/policy_mail_text.go:11]: up for discussion: language.Tag?

File: internal/api/grpc/admin/org.go [hunk 2]
```diff
@@ -65,15 +66,15 @@ func (s *Server) GetOrgIamPolicy(ctx context.Context, in *admin.OrgIamPolicyID)
 }
 
 func (s *Server) CreateOrgIamPolicy(ctx context.Context, in *admin.OrgIamPolicyRequest) (_ *admin.OrgIamPolicy, err error) {
-	policy, err := s.command.AddOrgIAMPolicy(ctx, orgIAMPolicyRequestToDomain(in))
+	policy, err := s.command.AddOrgIAMPolicy(ctx, authz.GetCtxData(ctx).OrgID, orgIAMPolicyRequestToDomain(in))
```
Comment (livio-a) [inline internal/api/grpc/admin/org.go:69]: OrgIamPolicyRequest has a required orgID field which would now be ignored

File: internal/api/grpc/admin/org.go [hunk 3]
```diff
@@ -65,15 +66,15 @@ func (s *Server) GetOrgIamPolicy(ctx context.Context, in *admin.OrgIamPolicyID)
...
 	if err != nil {
 		return nil, err
 	}
 	return orgIAMPolicyFromDomain(policy), err
 }
 
 func (s *Server) UpdateOrgIamPolicy(ctx context.Context, in *admin.OrgIamPolicyRequest) (_ *admin.OrgIamPolicy, err error) {
-	policy, err := s.command.ChangeOrgIAMPolicy(ctx, orgIAMPolicyRequestToDomain(in))
+	policy, err := s.command.ChangeOrgIAMPolicy(ctx, authz.GetCtxData(ctx).OrgID, orgIAMPolicyRequestToDomain(in))
```
Comment (livio-a) [inline internal/api/grpc/admin/org.go:77]: OrgIamPolicyRequest has a required orgID field which would now be ignored

File: internal/api/grpc/management/login_policy.go [hunk 4]
```diff
@@ -26,15 +26,15 @@ func (s *Server) GetDefaultLoginPolicy(ctx context.Context, _ *empty.Empty) (*ma
 }
 
 func (s *Server) CreateLoginPolicy(ctx context.Context, policy *management.LoginPolicyRequest) (*management.LoginPolicy, error) {
-	result, err := s.command.AddLoginPolicy(ctx, loginPolicyRequestToDomain(ctx, policy))
+	result, err := s.command.AddLoginPolicy(ctx, authz.GetCtxData(ctx).OrgID, loginPolicyRequestToDomain(ctx, policy))
```
Comment (livio-a) [inline internal/api/grpc/management/login_policy.go:29]: we should discuss how we want to pass the orgID / RO (separate param or in domain object)

**Review by livio-a — CHANGES_REQUESTED** [review]

File: internal/v2/domain/policy_mail_text.go (hunk 1 above)
Comment (fgerschwiler) [reply internal/v2/domain/policy_mail_text.go:11]: I just made exactly the same as we had before. in my opinion mailtexttype should also be a enum

File: internal/api/grpc/management/login_policy.go (hunk 4 above)
Comment (fgerschwiler) [reply internal/api/grpc/management/login_policy.go:29]: we have it every where as separate param so i thought we should make this here as well

**Review by livio-a — APPROVED** [review]

**Pull request merged**